boards/opencm904/include/periph_conf.h

```diff
@@ -60,13 +60,6 @@ extern "C" {
 /** @} */
 
 /**
- * @name ADC configuration
- * @{
- */
-#define ADC_NUMOF           (0)
```

Comment:
nope, related: the opencm904 uses a stm cpu and the reworked adc driver does not requite 'empty' numof defs anymore... So related cleanup.